marklogic/datadog_checks/marklogic/constants.py

```diff
@@ -10,7 +10,8 @@
 BASE_ENDPOINT = '/manage/v2'
 
 RESOURCE_TYPES = {
-    'cluster': {'plural': None, 'singular': 'cluster', 'tag_name': 'cluster_name'},
+    'cluster': {'plural': 'clusters', 'singular': 'cluster', 'tag_name': 'cluster_name'},
+    'clusters': {'plural': 'cluster', 'singular': 'cluster', 'tag_name': 'cluster_name'},
```

Comment:
Should this be
```suggestion
    'clusters': {'plural': 'clusters', 'singular': 'cluster', 'tag_name': 'cluster_name'},
```
instead?

Reply:
Yes! Also note this field isn't needed for `cluster` and `clusters` since we don't collect metrics for them, but I included it for consistency